calico-enterprise/network-policy/application-layer-policies/index.mdx

```diff
@@ -0,0 +1,11 @@
+---
+description: Use application layer policies to restrict ingress traffic based on HTTP attributes.
+hide_table_of_contents: true
+---
+
+# Application Layer Policies (ALP) using Envoy as DaemonSet
```

Comment:
Leave out implementation of titles. Suggest: Application layer policies to control ingress traffic

Reply:
Done